src/drivers/px4io/px4io_params.c

```diff
@@ -231,6 +231,126 @@ PARAM_DEFINE_FLOAT(PWM_MAIN_TRIM7, 0);
  */
 PARAM_DEFINE_FLOAT(PWM_MAIN_TRIM8, 0);
 
+/**
+ * Set the disarmed PWM for the main 1 output
+ *
+ * This is the PWM pulse the autopilot is outputting if not armed.
+ * When set to -1 the value for PWM_DISARMED will be used
+ *
+ * @reboot_required true
+ *
+ * @min -1
+ * @max 2200
+ * @unit us
+ * @group PWM Outputs
+ */
+PARAM_DEFINE_INT32(PWM_MAIN_DIS1, -1);
+
+/**
+ * Set the disarmed PWM for the main 2 output
+ *
+ * This is the PWM pulse the autopilot is outputting if not armed.
+ * When set to -1 the value for PWM_DISARMED will be used
+ *
+ * @reboot_required true
+ *
+ * @min -1
+ * @max 2200
+ * @unit us
+ * @group PWM Outputs
+ */
+PARAM_DEFINE_INT32(PWM_MAIN_DIS2, -1);
+
+/**
+ * Set the disarmed PWM for the main 3 output
+ *
+ * This is the PWM pulse the autopilot is outputting if not armed.
+ * When set to -1 the value for PWM_DISARMED will be used
+ *
+ * @reboot_required true
+ *
+ * @min -1
+ * @max 2200
+ * @unit us
+ * @group PWM Outputs
+ */
+PARAM_DEFINE_INT32(PWM_MAIN_DIS3, -1);
+
+/**
+ * Set the disarmed PWM for the main 4 output
+ *
+ * This is the PWM pulse the autopilot is outputting if not armed.
+ * When set to -1 the value for PWM_DISARMED will be used
+ *
+ * @reboot_required true
+ *
+ * @min -1
+ * @max 2200
+ * @unit us
+ * @group PWM Outputs
+ */
+PARAM_DEFINE_INT32(PWM_MAIN_DIS4, -1);
```

Comment:
These are also used for the fmu, so they need to be in the build, even if px4io is disabled.